fastlane/jetpack_metadata/ja/release_notes.txt

```diff
@@ -0,0 +1,3 @@
+ブラウザーに切り替える必要がある場合でも、 アプリの「自分」メニューにある QR コードをスキャンすることでログインできるようになりました。
+
+投稿のプレビュー時には、コンテンツの一部が切れることなく、縦向き表示と横向き表示を切り替えられるように なりました。
```

Comment:
Given those were new — i.e. translation for those seems to have come back after the last beta, contrary to most of other locales which already had a translation that you then amended to remove the first paragraph — I had a doubt if those were adjusted back when Team Global made their pass to amend all the copies, as opposed to having been translated by polyglots only afterwards and thus accidentally including the paragraph (given we kept it in the original to avoid making all translations fuzzy) in the translation.

But all seems good there in practice 👌  At least according to Google Translate, that copy doesn't include the paragraph about Insights (that was scoped out for this release and removed from other locales), so we're all good 👍